app/src/main/java/com/example/sharingang/ItemsListFragment.kt

```diff
@@ -24,46 +20,50 @@ class ItemsListFragment : Fragment() {
         viewModel.navigateToEditItem.observe(viewLifecycleOwner, { item ->
             item?.let {
                 this.findNavController().navigate(
-                        ItemsListFragmentDirections.actionItemsListFragmentToEditItemFragment(item)
+                    ItemsListFragmentDirections.actionItemsListFragmentToEditItemFragment(item)
                 )
                 viewModel.onEditItemNavigated()
             }
         })
         viewModel.viewingItem.observe(viewLifecycleOwner, {
             if (it) {
                 this.findNavController().navigate(
-                        ItemsListFragmentDirections.actionItemsListFragmentToDetailedItemFragment()
+                    ItemsListFragmentDirections.actionItemsListFragmentToDetailedItemFragment()
                 )
                 viewModel.onViewItemNavigated()
             }
         })
     }
 
     override fun onCreateView(
-            inflater: LayoutInflater, container: ViewGroup?,
-            savedInstanceState: Bundle?
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?

```

Comment:
Formatting looks off here, and otherwhere in the file

Reply:
Already formatted apparently